cmdx/config.go

```diff
@@ -75,6 +76,21 @@ func (c *Config) Load(cfg interface{}) error {
 	return nil
 }
 
+func (c *Config) LoadWithCmdBindFlag(cmd *cobra.Command, cfg interface{}) error {
+	v := viper.New()
```

Comment:
@mabdh any specific reason to do it with viper and not the config package loader?